R/helper_autoplot.R

```diff
@@ -0,0 +1,63 @@
+# reducing redundant code parts
+prepare_autoplot_cstf = function(task, resampling) {
+
+  data = task$data()
+  data$row_id = task$row_ids
+  data$indicator = 0
+  coords = task$coordinates()
+  coords$row_id = task$row_ids
+
+  if (grepl("Repeated", class(resampling)[1])) {
+    n_iters = resampling$iters / resampling$repeats(resampling$iters)
+  } else {
+    n_iters = resampling$iters
+  }
+
+  for (i in seq_len(n_iters)) {
+
+    row_id = resampling$instance$test[[i]]
+
+    data$indicator[data$row_id %in% row_id] = i
+  }
+  rm(resampling)
```

Comment:
This is usually not required.

Reply:
Good catch, no idea why it's there. Maybe a debugging leftover.